dev-python/pyamg/pyamg-4.0.0.ebuild

```diff
@@ -19,6 +19,7 @@ RDEPEND="
 	dev-python/numpy[${PYTHON_USEDEP}]
 	dev-python/pybind11[${PYTHON_USEDEP}]
 	dev-python/scipy[${PYTHON_USEDEP}]"
+BDEPEND="test? ( dev-python/pytest[${PYTHON_USEDEP}] )"
```

Comment:
can't you just try to use `distutils_enable_tests pytest` straight? pytest should find the tests too